### PR TITLE
(openssl.light) set path variable in install

### LINKED ISSUE #773: (openssl.light) path not set during choco install
## Expected Behavior
choco install adds an entry to the path to either the default install dir/bin or the custom install dir/bin

## Current Behavior
no path entry is made

## Possible Solution
Add an Install-ChocolateyPath

## Steps to Reproduce (for bugs)

1. cinst -y openssl.light
2. restart shell
3. write-host $env:path 
4. no entry for openssl

## Context
you have to manually add the path to use openssl via the cli

## Your Environment

* Package Version used: 1.1.0.50000000
* Operating System and version: windows 7
* Chocolatey version:
* Install/uninstall gist:

### PR DESCRIPTION
**Pull request by chrisevett**
## Description
Added a Install-ChocolateyPath to add a path entry for the openssl bin directory

## Motivation and Context
fixes #773

## How Has this Been Tested?
windows 7 testing using choco install -fdv .\openssl.light.nuspec

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package have been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this mean usually the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this mean usually the notes in the description of a package).
- [x] All files is up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/.github/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/openssl.light/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -20,5 +25,7 @@ $packageArgs = @{
 }
 
 Install-ChocolateyInstallPackage @packageArgs
+$binPath = "$($dir)\bin" 
+Install-ChocolateyPath -PathToInstall $binPath
```
Comment (AdmiringWorm) [inline automatic/openssl.light/tools/chocolateyInstall.ps1]: We would need to add a check here to see if the bindir actually exist, the user may have overridden the install directory.
Best option would be to use the [Get-AppInstallLocation](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/extensions/chocolatey-core.extension/extensions/Get-AppInstallLocation.ps1) available from the `chocolatey-core.extenion` package (which we already have a dependency on), and get the installed directory that way (of course, it needs to be tested that it actually exist).

Comment (chrisevett) [inline automatic/openssl.light/tools/chocolateyInstall.ps1]: Thanks for the input. These are good suggestions. I'll send updates tomorrow 💯 💯 💯

Comment (chrisevett) [inline automatic/openssl.light/tools/chocolateyInstall.ps1]: Just spitballing, is this the right way to add to the path variable in a choco package?

Comment (AdmiringWorm) [inline automatic/openssl.light/tools/chocolateyInstall.ps1]: 👍 
Another note, if you go with the `Get-AppInstallLocation` (which would be preferred), you can most likely just reuse the `softwareName` used in the `$packageArgs` variable.

Comment (AdmiringWorm) [inline automatic/openssl.light/tools/chocolateyInstall.ps1]: > Just spitballing, is this the right way to add to the path variable in a choco package?

I've not used the `Install-ChocolateyPath` that often myself, so I can't remember exactly.
But yes, I believe so.

Comment (chrisevett) [inline automatic/openssl.light/tools/chocolateyInstall.ps1]: Gotcha. Thanks again for the direction.

**Comment by chrisevett** [comment]
@majkinetor made the change you suggested and @AdmiringWorm made the changes you suggested as well. Let me know if you have any further feedback.

**Comment by AdmiringWorm** [comment]
The changes looks good to me, I just need to test it out with a custom path first, and then it can be merged.

**Comment by AdmiringWorm** [comment]
@chrisevett everything works great, thanks for your contribution

**Pull request merged**